website/docs/sdk-reference/java.md

```diff
@@ -20,10 +20,14 @@ Maven:
   <version>[7.0.0,)</version>
 </dependency>
 ```
-Gradle:
+Old Gradle versions:
 ```bash
 compile group: 'com.configcat', name: 'configcat-java-client', version: '7.+'
 ```
+Gradle 7 and above:
+```bash
+implementation group: 'com.configcat', name: 'configcat-java-client', version: '7.+'
```

Comment:
```suggestion
implementation "com.configcat:configcat-java-client:7.+"
```

Maybe the shorthand version is a bit better, I'm going to change this in the java repo's examples & docs as well